test/llvm/macros/complex-macro3.good

```diff
@@ -0,0 +1 @@
+complex-macro3.chpl:1: error: 'c_int' undeclared (first use this function)
```

Comment:
Is this the right error message?

Reply:
Maybe `ComplexVar` not being a `var` is the issue?

Reply:
Yeah this test isn't quite right yet